{{cookiecutter.project_shortname}}/tests/requirements.txt

```diff
@@ -3,10 +3,9 @@
 # pip install -r requirements.txt
 
 chromedriver-binary
-dash>=0.38.0
+dash[testing]>=1.0.2
 ipdb
 percy
```

Comment:
same here: `percy` adn `selenium` will be installed by `dash[testing]`

Reply:
For that matter I don't know why we need *anything* other than `dash[testing]`... none of our other repos require `chromedriver-binary`, and I don't see `ipdb`, `flake8`, or `pylint` used - do you see any reason we should keep those?

Reply:
if not needed, I'm fine to 🔪